app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -543,6 +551,41 @@ public void onClick(View v) {
         }
     }
 
+    /**
+     * Debug method to toast a view's ID whenever it is clicked.
+     */
+    private void setClickListenersForEverything() {
```

Comment:
This seems potentially useful, but I'd put it into a view utility/instrumentation class statically